src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const calls = (a) => {
```

Comment:
why did you call it like this? I guess there should be a different name for this function